.github/workflows/docker-build-push.yaml

```diff
@@ -0,0 +1,54 @@
+name: docker-build-push
+
+on:
+  workflow_dispatch:
+
+  push:
+    branches:
+      - 'unstable'
+
+jobs:
+  docker-build-push:
+    strategy:
+      matrix:
+        go-version: [ 1.17.x ]
```

Comment:
@ColinChamber Do we need to update the Go version to 1.19？

Reply:
Yeah, you need to modify the version of golang used to build the image, [here](https://github.com/xiao-jay/kvrocks-operator/blob/build-image-ci/Dockerfile#L4).  Does docker runtime environment require a specific version of golang?

Reply:
done